### PR TITLE
subway map, nf-core samtools modules, deepvariant gpu support

### PR DESCRIPTION
**Pull request by yuukiiwa**
## Remaining tasks
- [x] update subway map (assigned to Chris)
- [x] Add gpu support for deepvariant (assigned to Chris)
- [x] Switch back to using the nf-core modules for samtools (assigned to Yuk Kei)

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
    - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanoseq/tree/master/.github/CONTRIBUTING.md)
    - [x] If necessary, also make a PR on the nf-core/nanoseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by christopher-hakkaart** [comment]
GPU has been added but I want to test it on another system before merging

**Comment by yuukiiwa** [comment]
Thanks, @christopher-hakkaart!! That's great!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,20 +38,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-| `deepvariant`               |             | 1.0.3       |
-| `jaffa`                     |             | 2.0         |
-| `m6anet`                    |             | 1.0         |
-| `medaka`                    |             | 1.4.4       |
-| `multiqc`                   | 1.10.1      | 1.11        |
-| `ont_fast5_api`             |             | 4.0.0       |
-| `pepper_margin_deepvariant` |             | 0.8         |
-| `sniffles`                  |             | 1.0.12      |
-| `xpore`                     |             | 2.1         |
+| Dependency                      | Old version | New version |
```
Comment (yuukiiwa) [inline CHANGELOG.md:41]: need to add the following row:
```
| `samtools`                      | 1.14        | 1.15        |
```

File: nextflow_schema.json [hunk 2]
```diff
@@ -228,6 +228,11 @@
                     "description": "Specifies if MNPs will be split into SNPs when using medaka.",
                     "fa_icon": "fas fa-file-alt"
                 },
+                "deepvariant_gpu": {
+                    "type": "boolean",
+                    "fa_icon": "fas fa-exchange-alt",
+                    "description": "Specifies whether to call variants with pepper_margin_deepvariant in GPU mode.."
```
Comment (christopher-hakkaart) [inline nextflow_schema.json:234]:
```suggestion
                    "description": "Specifies whether to call variants with pepper_margin_deepvariant in GPU mode."
```

**Review by christopher-hakkaart — APPROVED** [review]
One typo (by me!) but otherwise great.

**Pull request merged**